### ISSUE TITLE
Rotary encoders

### ISSUE THREAD

**Issue by alexarje** [alexarje/MultiControl#5]
My project is a DIY-Midi-Controller to control Ableton Live with it.
I am using the Bu0386x-Controller by Leo Bodnar:
http://www.leobodnar.com/shop/index.php?main_page=product_info&cPath=67&products_id=180

I want to use 9 rotary encoders:
-8 for controlling filter bands
-1 for scrolling through the ableton scenes ( I assume you know what Ableton Live is;-) )

I am afraid that it is impossible to use rotary encoders in Multi Control because:

When I turn a rotary encoder, Multi Control indicates low/high per detent; just as if I was pressing a Button (per detent).

To sum it up: I could manage to assign a rotary encoder for my purposes considering Ableton Live.

Do you have any experience with rotary encoders in Multi Control?
